### DISCUSSION TITLE
Re-order settings menus

### DISCUSSION THREAD

**Discussion by TobiPeterG** [osmandapp/OsmAnd#20923] [Ideas]
While it's great that OSMand is very customizable, however the settings menus for the elements can be very confusing. I find myself often looking for settings in 3 different places before finding them.
It would be great if
A) every setting could be reached from one main menu (that shouldn't be the sidebar as that doesn't comply with the current design language of Android)
B) the settings menus all have a consistent look-and-feel (e.g. not some elements having an icon and some not)
C) settings about the same thing/related are merged or linked to

**Comment by DmitryAlexei** [comment]
All settings related to a specified profile can be found via menu > settings > profile
